src/tusb_option.h

```diff
@@ -75,17 +75,21 @@
 #define OPT_MCU_CXD56             400 ///< SONY CXD56
 
 // TI MSP430
-#define OPT_MCU_MSP430x5xx    500 ///< TI MSP430x5xx
+#define OPT_MCU_MSP430x5xx        500 ///< TI MSP430x5xx
```

Comment:
Just a thought ...

You might consider changing this to a union, unless there are benefits I am not aware of to leaving as `#define`.

e.g.:
```C
enum {
    OPT_MCU_NONE = 0,
    OPT_MCU_LPC11UXX = 1,
    /* etc. */
} /* OPT_MCU_Enum */ ;

Why?
* You get the same compile-time values, but you've now added type information -- so debuggers can print the friendly name when it's used.
* Values can still be used in C++11 (and later) `constexpr`, just like `#define`
* If you give the enum a name, function / template parameters can specify the type, improving usage and finding more errors at compile-time 

Obviously, this is optional, but does result in a "friendlier" library.

Reply:
it is complicated, trust me I tried to use enum wherever I could. just leave that one for now.